### ISSUE TITLE
"Sharing internet" without VPN

### ISSUE THREAD

**Issue by mirko** [freifunk-berlin/firmware#212]
tl;dr: Don't enforce an VPN when deciding to 'Share internet'.

I will try to haul off as little as possible: I'm pretty sure not allowing to just share your plain internet connection without VPN was more a political rather than a technical decision.
However the situation is changing: there are statements in favour of us, also the 'ISP privilege' got interpreted in favour of us.
Although the VPN was/is a nice workaround, it's not the solution. And it's not Freifunk (if it doesn't have to be). I'd like to encourage people to 'expose' themselves, since the risk of getting sued gets smaller and smaller. Hiding is not necessary anymore.
I'd herewith suggest adding an option for sharing the internet without needing an VPN. Displaying a big warning about potential consequences would probably make sense though :)

**Comment by andrenarchy** [comment]
+1 for making the VPN optional!

**Comment by booo** [comment]
Related to https://github.com/freifunk-berlin/firmware/issues/144

**Comment by melle** [comment]
Would it be sufficient to revert patches/011-luci-freifunk-policyrouting-berlin.patch in case I want to brew my own binary without vpn?

**Comment by cholin** [comment]
The easiest way is to add your default route to the `olsr-tunnel` routing table. You may use @booo's [rudimentary hotplug script](https://github.com/freifunk-berlin/firmware/issues/144) for that.

**Comment by sarumpaet** [comment]
Duplicate of #144

**Comment by SvenRoederer** [comment]
see also https://github.com/freifunk-berlin/firmware/issues/455

**Comment by SvenRoederer** [comment]
I outlined a solution in https://github.com/freifunk-berlin/firmware-packages/pull/125

**Comment by SvenRoederer** [comment]
this will be implemented by https://github.com/freifunk-berlin/firmware/pull/498 and https://github.com/freifunk-berlin/firmware-packages/pull/125

And it's tested to work by independent testers

**Comment by SvenRoederer** [comment]
merged in 8664d2c after testing of at least 2 moths
